### PR TITLE
Add tests for CTA work, revert lost view application format

### PR DESCRIPTION
**Pull request by arthurashman**
## Description of change
Added tests to ensure CTAs show on correct states, and status banners are correct per status. Also improving to ensure buttons have correct copy when assigned/unassigned/assigned to another 

Designs for status badge/assignement/dates https://www.figma.com/file/TELZW9jdNqFcrI4yZoH7BV/Crime-Review?node-id=4491%3A16813&t=hAjiuqIqCfLVzKtP-1
 
### Before changes:
image

### After changes:
image

### CODE REVIEW

File: app/views/crime_applications/_current_assignment.html.erb [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-    <% if crime_application.current_assignment.user_id == current_user_id %>
-    <%= button_to t('calls_to_action.unassign_from_self'),
-                  assigned_application_path(crime_application),
-                  method: :delete,
-                  class: 'govuk-button govuk-button--secondary' %>
+  <dd class="govuk-summary-list__value govuk-!-width-one-half govuk-!-padding-0 govuk-body-l">
+    <div class="govuk-row">
+      <strong><%= t(:assigned_to, scope: 'crime_applications.labels') %></strong>
+      <% if crime_application.current_assignment.nil? %>
+        <%= 'Unassigned' %>
```
Comment (willmcb) [inline app/views/crime_applications/_current_assignment.html.erb:6]: Not sure this needs to be in an erb <%= as it is a string literal

**Review by willmcb — APPROVED** [review]
LGTM - one tiny non-blocking comment :)

File: app/views/crime_applications/_current_assignment.html.erb (hunk 1 above)
Comment (arthurashman) [reply app/views/crime_applications/_current_assignment.html.erb:6]: Yep, meant to change that - that's how I found it

Comment (arthurashman) [reply app/views/crime_applications/_current_assignment.html.erb:6]: Wanted to put it into en.local

**Pull request merged**